src/config.h-example

```diff
@@ -1,6 +1,9 @@
 /* Example for a config file for the MQTT433gateway. Adapt as needed and copy to net-passwd.h
```

Comment:
File name has changed, but the header still refers to `net-passwd.h`

Reply:
Good catch. It should be `config.h`. I will fix this.

Thank you!